data/helpers.d/string

```diff
@@ -18,8 +18,18 @@ ynh_string_random() {
 # | arg: target_file - File in which the string will be replaced.
 ynh_replace_string () {
 	delimit=@
-	match_string=${1//${delimit}/"\\${delimit}"}	# Escape the delimiter if it's in the string.
+	# Escape the delimiter if it's in the string.
+	match_string=${1//${delimit}/"\\${delimit}"}
 	replace_string=${2//${delimit}/"\\${delimit}"}
+
+        # Escape any backslash to preserve them as simple backslash.
+        match_string=${match_string//\\/"\\\\"}
+        replace_string=${replace_string//\\/"\\\\"}
```

Comment:
I think it creates a problem if the password contains a @ because it will be transformed in \@ and here in \\@ => this will create an error...

Reply:
You're right, there's a regression here.
It's fixed now.